src/sentry/api/serializers/models/project.py

```diff
@@ -1005,6 +1006,16 @@ def serialize(
                 },
             }
         )
+
+        if features.has("organizations:metrics-extrapolation", obj.organization):
```

Comment:
@obostjancic @shellmayr @vgrozdanic
Could you please clarify how we are guarding project properties? The current implementation won't work because the organization is a string, which isn't what we need. Could you point me to the correct way to check for an organization feature flag in the project?

Reply:
So since `obj` is Project, `obj.organization` should be an Organization model, so this should work.

Reply:
i thought the same but when testing it locally I do not get an object 🤔